components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/util/ClaimConstants.java

```diff
@@ -132,4 +136,36 @@ public enum ClaimUniquenessScope {
         WITHIN_USERSTORE,
         ACROSS_USERSTORES
     }
+
+    /**
+     * Enum for shared profile value resolving methods.
+     */
+    public enum SharedProfileValueResolvingMethod {
+        FROM_SHARED_PROFILE("FromSharedProfile"),
+        FROM_ORIGIN("FromOrigin"),
+        FROM_FIRST_FOUND_IN_HIERARCHY("FromFirstFoundInHierarchy");
+
+        private final String name;
+
+        SharedProfileValueResolvingMethod(String name) {
+
+            this.name = name;
+        }
+
+        public String getName() {
+
+            return name;
+        }
+
+        // Method to match resolving method names with enum constants.
```

Comment:
Method comment format

Reply:
fixed in [9148662](https://github.com/wso2/carbon-identity-framework/pull/6247/commits/9148662693d40927702ebe53b2250979a60d8c6f)